### PR TITLE
Add welcome message to features

### LINKED ISSUE #7277: Add welcoming message as a feature
The welcoming text currently is fixed and not editable by the admins.Moving the welcoming text into a feature will allow site admins to edit it according to their needs.

Follow up to #6096 implemented by pull request #7236 

**Please describe the desired behavior.**
Admins can now edit the welcoming message present here.
Screenshot from 2020-01-16 11-53-22

**Additional context (optional)**
Check out the discussion here:-
https://github.com/publiclab/plots2/pull/7236#issuecomment-575261143

### PR DESCRIPTION
**Pull request by Tlazypanda**
Fixes #7277 

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by Tlazypanda** [comment]
@jywarren not sure if these are all the changes required :sweat_smile: ..can you help review this and point in the right direction? Thanks :smile:

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -39,6 +39,9 @@
         %>
         <% if is_new_contributor %>
           <div id="new_contributor_msg">
+            <% cache('feature_welcome-msg', skip_digest: true) do %>
+            <%= feature("welcome-msg") %>
+            <% end %>
             <p><i class="fa fa-hand-peace-o" aria-hidden="true"></i>
             <%= author %> is a new contributor. Take care in
```
Comment (jywarren) [inline app/views/comments/_form.html.erb]: So, actually we can remove the message you've added, and I can add it directly to the database so that it becomes the current feature text. Then we can merge your code and next time it's published, it'll pick up the text!

**Comment by Tlazypanda** [comment]
@jywarren is this alright?

**Comment by Tlazypanda** [comment]
@jywarren the travis failure seems to be unrelated to my changes can you kindly check?

**Comment by cesswairimu** [comment]
Restarting travis

**Comment by Tlazypanda** [comment]
@jywarren not sure why the tests are failing can you help me out?Thanks :v: Not really sure if this is how the feature is to be implemented :sweat_smile:

**Comment by jywarren** [comment]
Huh. Very odd! I can't seem to get it to pass - you're totally right it's unrelated. Let me try again... sorry!!!

**Comment by Tlazypanda** [comment]
@jywarren Ohh :sweat_smile: no issues :smile:  any workaround for this ?

**Comment by jywarren** [comment]
☹️ I'm not sure! I tried retriggering again. If that doesn't work, would you mind rebasing this over the latest master branch?

**Comment by jywarren** [comment]
Hooray!!! It worked! Sorry this was so bad!

🎉 ❤️ 💥

**Comment by jywarren** [comment]
Thanks for your patience!!!
